src/lib/Components/Consignments/Components/ImageSelection.tsx

```diff
@@ -64,19 +81,25 @@ const TakePhotoImage = (props: TakePhotoImageProps) =>
   </TouchableOpacity>
 
 const ImageForURI = (props: ImagePreviewProps) =>
-  <TouchableHighlight
-    onPress={() => props.onPressItem(props.data.image.uri)}
+  <View
     style={{
-      backgroundColor: colors["gray-regular"],
+      borderWidth: 2,
+      borderColor: props.selected ? "white" : null,
+      margin: 4,
       height: 158,
       width: 158,
-      margin: 4,
-      borderColor: props.selected ? "white" : null,
-      borderWidth: 2,
     }}
   >
-    <Image source={{ uri: props.data.image.uri }} style={{ height: 154, width: 154 }} />
-  </TouchableHighlight>
+    <TouchableHighlight
+      onPress={() => props.onPressItem(props.data.image.uri)}
+      style={{
+        opacity: props.selected ? 0.5 : 1.0,
+      }}
+    >
+      <Image source={{ uri: props.data.image.uri }} style={{ height: 154, width: 154 }} />
+    </TouchableHighlight>
+    {props.selected ? <SelectedIcon /> : <View />}
```

Comment:
can this be done nicer?

Reply:
Barely: `props.selected || <SelectedIcon />`, kinda `props.selected ? <SelectedIcon /> : null` 

`null` is a better call than view, but whether the `||` is too far is subjective

Reply:
`null` was the thing I was looking for. Duh! Will update :)